src/instance.rs

```diff
@@ -39,11 +39,10 @@ impl<T> Py<T> {
     pub fn new(py: Python, value: impl Into<PyClassInitializer<T>>) -> PyResult<Py<T>>
```

Comment:
I wonder if we should remove this method and instead always encourage use of `PyCell::new()`.

Or maybe the argument type should be `impl IntoPy<PyObject>`, so that e.g. `Py<PyTuple>` can also be created with `Py::new()`.

Reply:
:+1: